.mailmap

```diff
@@ -102,3 +102,5 @@ Zev Benjamin <zev@zulip.com> <zev@mit.edu>
 Zixuan James Li <p359101898@gmail.com>
 Zixuan James Li <p359101898@gmail.com> <39874143+PIG208@users.noreply.github.com>
 Zixuan James Li <p359101898@gmail.com> <359101898@qq.com>
+Joseph Ho <josephho678@gmail.com>
+Joseph Ho <josephho678@gmail.com> <62449508+Joelute@users.noreply.github.com>
```

Comment:
That little red icon shows you don't have a newline at the end of the file -- most editors will auto-add them, and that's good practice. I'm a bit surprised our linter didn't flag this; I guess this file must not be included in what it checks for missing newlines at EOF.

Reply:
Just pushed another commit. I've also tweaked my editor to auto add them. Kind of curious on the reason for adding a newline at the end of the file. Is it just a code style thing?

Reply:
Here's a good explanation of why all lines (including the last one) end with a newline:
https://stackoverflow.com/a/729795
(Probably #learning would be the best place for any further discussion — it'll be interesting for other people too.)